src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -34,44 +44,45 @@ export class CaseViewerComponent implements OnInit, OnDestroy, AfterViewInit {
   static readonly space = '%20';
 
   @Input()
-  hasPrint = true;
+  public hasPrint = true;
   @Input()
-  hasEventSelector = true;
-
-  BANNER = DisplayMode.BANNER;
-
-  caseDetails: CaseView;
-  sortedTabs: CaseTab[];
-  caseFields: CaseField[];
-  error: any;
-  triggerTextStart = CaseViewerComponent.TRIGGER_TEXT_START;
-  triggerTextIgnoreWarnings = CaseViewerComponent.TRIGGER_TEXT_CONTINUE;
-  triggerText: string = CaseViewerComponent.TRIGGER_TEXT_START;
-  ignoreWarning = false;
-  activitySubscription: Subscription;
-  caseSubscription: Subscription;
-  errorSubscription: Subscription;
-  dialogConfig: MatDialogConfig;
-
-  callbackErrorsSubject: Subject<any> = new Subject();
+  public hasEventSelector = true;
+
+  public BANNER = DisplayMode.BANNER;
+
+  public caseDetails: CaseView;
+  public sortedTabs: CaseTab[];
+  public caseFields: CaseField[];
+  public formGroup: FormGroup;
+  public error: any;
+  public triggerTextStart = CaseViewerComponent.TRIGGER_TEXT_START;
+  public triggerTextIgnoreWarnings = CaseViewerComponent.TRIGGER_TEXT_CONTINUE;
+  public triggerText: string = CaseViewerComponent.TRIGGER_TEXT_START;
+  public ignoreWarning = false;
+  public activitySubscription: Subscription;
+  public caseSubscription: Subscription;
+  public errorSubscription: Subscription;
+  public dialogConfig: MatDialogConfig;
+
+  public callbackErrorsSubject: Subject<any> = new Subject();
   @ViewChild('tabGroup') public tabGroup: MatTabGroup;
 
   constructor(
-    private ngZone: NgZone,
-    private route: ActivatedRoute,
-    private navigationNotifierService: NavigationNotifierService,
-    private orderService: OrderService,
-    private activityPollingService: ActivityPollingService,
-    private dialog: MatDialog,
-    private alertService: AlertService,
-    private draftService: DraftService,
-    private caseNotifier: CaseNotifier,
-    private errorNotifierService: ErrorNotifierService,
+    private readonly ngZone: NgZone,
+    private readonly route: ActivatedRoute,
+    private readonly navigationNotifierService: NavigationNotifierService,
+    private readonly orderService: OrderService,
+    private readonly activityPollingService: ActivityPollingService,
+    private readonly dialog: MatDialog,
+    private readonly alertService: AlertService,
+    private readonly draftService: DraftService,
+    private readonly caseNotifier: CaseNotifier,
+    private readonly errorNotifierService: ErrorNotifierService,
     private readonly location: Location
   ) {
   }
 
-  ngOnInit() {
+  public ngOnInit(): void {
```

Comment:
I am not really a fan of putting accessor declarations on implemented methods, it goes against the standard Angular coding style, any reason for this?

Reply:
My understanding is that putting _explicit_ `public` accessors on implicitly public methods - including these ones - is part of the `tslint` standard. We have a bit of a disparity between MC and the toolkit in this regard but MC flags it as a warning and I think we should be consistent across all the projects we're responsible for. 🤷‍♂️

Reply:
Where is the Angular coding style guide that says we shouldn't do that? Happy to be proven wrong on this one but I think `tslint` flags it as a warning within MC if it's missing...

Reply:
Reverted, pending a conversation in the Tech Forum about what we _should_ be doing as there's enormous inconsistency across within each project and across all the projects.